src/rsz/src/Resizer.cc

```diff
@@ -565,6 +565,172 @@ void Resizer::balanceRowUsage()
 
 ////////////////////////////////////////////////////////////////
 
+// Inspect the timing arcs on a buffer size to find the capacitance
+// points on the piecewise linear delay model
+static void populateBufferCapTestPoints(LibertyCell* cell,
+                                        LibertyPort* in,
+                                        LibertyPort* out,
+                                        std::vector<float>& points)
```

Comment:
Any reason not to use a map here rather than a vector + sort + unique?

Reply:
Well map is too much (we have nothing interesting to put in as values), but `std::set<float>` might be fine. At first I wasn't sure whether comparison of floats is robust enough, but Internet says it's OK to use as a key